index.js

```diff
@@ -217,7 +225,7 @@ function findBestSubtitleFile(list) {
     .filter((lang) => lang.toString().substring(0,2) === 'en')
     .map((lang) => ({
       lang,
-      subs: list[lang].find((x) => x.ext === 'vtt'),
+      subs: list[lang].find((x) => x.ext === 'vtt' && x.protocol !== 'm3u8_native'),
```

Comment:
Playlists marked as 'm3u8_native' do not work when we feed it into gstreamer. 

For some videos, if the m3u8_native option is the first one available, we end up picking it and it doesn't work. e.g. https://vimeo.com/710964142 has subtitles but not when played in the vivi app. 

This won't affect youtube videos, they provide a m3u8_native version and a normal vtt version.